styles.css

```diff
@@ -96,3 +99,248 @@ nav {
 .social-media > a:hover {
   border-bottom: 1px solid #ff6b00;
 }
+
+.works {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  background: #fff;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.titel {
+  margin: 0 auto;
+  margin-top: 120px;
+  margin-bottom: 24px;
+  width: 90%;
+  font-family: "Crete Round", serif;
+  font-size: 40px;
+  color: #1c1a19;
+}
+
+hr {
+  margin: 0 24px;
+  margin-bottom: 80px;
+}
+
+.placeholder {
+  width: 327px;
+  height: 237px;
+}
+
+.post {
+  width: 327px;
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+  margin-top: 24px;
+  margin-bottom: 12px;
+}
+
+.info1 {
+  width: 327px;
+  font-weight: 400;
+  font-family: "Inter", sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  margin-bottom: 12px;
+}
+
+.categories {
+  display: flex;
+  width: 100%;
+  gap: 12px;
+  padding-bottom: 20px;
+}
+
+.categories > li {
+  font-size: 15px;
+  font-family: "Inter", sans-serif;
+  color: #091e42;
+  list-style-type: none;
+  padding: 10px 12px;
+  background: #fff;
+  border: 1px solid #8993a4;
+}
+
+.btn {
+  color: #fff;
+  align-self: flex-start;
+  background-color: #ff6b00;
+  padding: 12px;
+  font-weight: 700;
+  font-size: 17px;
+  font-family: "Inter", sans-serif;
+  border: none;
+}
```

Comment:
- [ ] Please add the styling for the other states of the buttons as well(`active`, `hover` and `disabled`), as they are specified in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design. 😁